example/src/index.ts

```diff
@@ -36,18 +36,22 @@ container.style.width = styleConfig.mainWidth + "px";
 const testDataProvider = new TestDataProvider(styleConfig.mainWidth);
 let timeGraph = testDataProvider.getData({});
 const unitController = new TimeGraphUnitController(timeGraph.totalLength);
-unitController.numberTranslator = (theNumber: number) => {
-    const milli = Math.floor(theNumber / 1000000);
-    const micro = Math.floor((theNumber % 1000000) / 1000);
-    const nano = Math.floor((theNumber % 1000000) % 1000);
-    return milli + ':' + micro + ':' + nano;
+unitController.numberTranslator = (theNumber: bigint) => {
+    let num = theNumber.toString();
+    if (num.length > 6) {
+        num = num.slice(0, -6) + ':' + num.slice(-6);
+    }
+    if (num.length > 3) {
+        num = num.slice(0, -3) + ':' + num.slice(-3);
+    }
+    return num;
 };
 
 const providers = {
     dataProvider: (range: TimelineChart.TimeGraphRange, resolution: number) => {
         const length = range.end - range.start;
-        const overlap = ((length * 20) - length) / 2;
-        const start = range.start - overlap > 0 ? range.start - overlap : 0;
+        const overlap = BigInt(((length * BigInt(20)) - length) / BigInt(2));
```

Comment:
Not to mention that length*20-length = length*19, which makes an overlap of length*9.5. Since this is just a test application, I'll just change it to length*10 because it will make our brainz hurt less.

Reply:
Hey cool, github changes asterisks to italic formatting